pkgs/servers/sql/mariadb/default.nix

```diff
@@ -181,6 +182,17 @@ everything = stdenv.mkDerivation (common // {
     sed -i 's/-mariadb/-mysql/' "$out"/bin/galera_new_cluster
   '';
 
+  postFixup =
+    let
+      mytopEnv = perl.withPackages (p: with p; [ DataDumper DBDmysql DBI TermReadKey ]);
+    in
+      # perlPackages.DBDmysql is broken on darwin
+      optionalString (!stdenv.isDarwin) ''
+        sed -i 's|/usr/bin/perl|${mytopEnv.interpreter}|' $out/bin/mytop
```

Comment:
Use `patchShebangs` here by adding mytopEnv to `buildInputs`.

Reply:
Also using `wrapProgramInPlace` from `makeWrapper` to extend `PATH` is the better option to add `less` and `more` over sed. Looking at the code only either less or more is required. I would go for less.

Reply:
Thank you very much @Mic92! How does this newest revision look?